src/website/views/components/forms/search-input/_examples.pug

```diff
@@ -0,0 +1,193 @@
+h2 Examples
+p.-text
+  | To render a search input, apply the class <code>chi-search__input</code> to an <code>input type="search"</code>.
```

Comment:
Isn't this HTML blueprint specific message? I think for it's irrelevant for web-component tab